testutil/network/network.go

```diff
@@ -157,9 +158,10 @@ type (
 		ValAddress sdk.ValAddress
 		RPCClient  tmclient.Client
 
-		tmNode *node.Node
-		api    *api.Server
-		grpc   *grpc.Server
+		tmNode  *node.Node
+		api     *api.Server
+		grpc    *grpc.Server
+		grpcWeb *http.Server
```

Comment:
Why do we need grpc-web for testnets? AFIAIK, the only reason we have this is for it to play nicely with Javascript-based clients.

Reply:
That's true. This is just for tests.